pygmo/docstrings.cpp

```diff
@@ -1087,7 +1087,7 @@ The best known solution for the )"
 
 std::string translate_docstring()
 {
-    return R"(__init__(udp = null_problem(), translation = [0.])
+    return R"(__init__(udp, translation)
```

Comment:
We do allow default values, it's also written in the documentation below, and it mirrors what happens in C++ as well. I don't see a reason to hide this fact from the user.